aim/sdk/__init__.py

```diff
@@ -1,18 +1,4 @@
 # Main functions
 from aim.sdk.track import track, set_params
 from aim.sdk.flush import flush
-from aim.sdk.init import init
```

Comment:
`init` method should be kept, it used to create a default session for forwarding `aim.track` to `default_session.track`.

Reply:
It was also used for specifying experiment name and repo path of default session.

Reply:
should we stub `init` method or should we keep this behavior? as of now, `Repo` (and legacy `Session`) do not require special initialization before usage.  the only thing required is `aim init` being ran for that repo.